### DISCUSSION TITLE
Limited Maintenance (4/22 - 5/22)

### DISCUSSION THREAD

**Discussion by rmorshea** [idom-team/idom#708] [Announcements]
@rmorshea will be on vacation from April 22 to May 22 of 2022 and will most likely be unable to respond to, or resolve issues during this period. If you need help, feel free to reach out to others in the community:

- [acivitillo](https://github.com/acivitillo)
- [Archmonger](https://github.com/Archmonger)

**Comment by acivitillo** [comment]
so the month of April? Will someone be able to review/approve PRs or do we keep them on standby?

**Reply by rmorshea** [reply]
April 22 to May 22, so a month beginning at the end of April.

I will try to take a look at any PRs people contribute, but if they aren't trivial (e.g. docs, minor improvements, simple bug fixes), they will probably have to wait until I'm back and can do a more thorough review. There definitely will not be any releases during this period though.

**Reply by acivitillo** [reply]
enjoy your holidays!

**Comment by rmorshea** [comment]
Outside of small fixes, the upcoming 0.38 release will likely be the last release before my vacation.

**Comment by rmorshea** [comment]
I'm back! I will slowly start ramping up work here again.
